contrib/mesos/docs/ha.md

```diff
@@ -1,4 +1,25 @@
-## High Availability
+# High Availability
+
+Kubernetes on Mesos supports 2 HA modes:
+
+* [Hot-standby](#hot-standby)
+* [Cold-standby](#cold-standby)
+
+In the hot-standby mode, all master components (apiserver, controller manager, and scheduler) actively run 
```

Comment:
This just reads in a fashion that's too misleading. "Kubernetes on Mesos supports 2 HA modes" and then each mode is described. We don't have hot-standby support integrated into the k8sm controller manager so this sentence isn't accurate. I understand you're probably trying to describe the intended design. But the implementation just isn't there yet and I don't want users wondering why hot-standby doesn't work with controller-manager when the docs say that it's supported.